<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieDialog</class>
 <widget class="QDialog" name="MovieDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>289</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>396</width>
    <height>289</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Movie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/movie64.png</normaloff>:/Icons/movie64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modeCombo">
       <property name="toolTip">
        <string>Some like it hot.</string>
       </property>
       <item>
        <property name="text">
         <string>Forward</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Backward</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CanvasSliderWidget" name="csWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 100, 0);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid blue;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Current frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="currentFrameMinus10button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="currentFrameMinus1button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentFrameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="currentFramePlus1button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="currentFramePlus10button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid blue;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Frame interval (ms)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="intervalMinus10button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="intervalMinus1button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="intervalLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>99999</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="intervalPlus1button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="intervalPlus10button">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid blue;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Skip factor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid blue;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Starting frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_17">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_18">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_19">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_20">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid blue;</string>
        </property>
        <property name="text">
         <string>End frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_21">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_22">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_23">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_24">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasSliderWidget</class>
   <extends>QWidget</extends>
   <header>MovieDialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
